### PR TITLE
concept intersect internal

### PR DESCRIPTION
**Pull request by catalamarti**

### CODE REVIEW

File: R/addConceptIntersect.R [hunk 1]
```diff
@@ -92,99 +79,168 @@ addConceptIntersect <- function(x,
...
+
+  # concepts table
+  conceptsTable <- getConceptsTable(conceptSet)
+  nm <- omopgenerics::uniqueTableName(tablePrefix)
+  cdm <- omopgenerics::insertTable(cdm = cdm, name = nm, table = conceptsTable)
+
+  # ids
+  conceptSetId <- conceptSetId(conceptSet)
+
+  conceptSet <- list("mc" = 4112343)
```
Comment (edward-burn) [inline R/addConceptIntersect.R:123]: not sure why this is here?

File: tests/testthat/setup.R [hunk 2]
```diff
@@ -1,3 +1,9 @@
+withr::local_envvar(
+  R_USER_CACHE_DIR = tempfile(),
+  .local_envir = teardown_env(),
+  EUNOMIA_DATA_FOLDER = Sys.getenv("EUNOMIA_DATA_FOLDER", unset = tempfile())
+)
+CDMConnector::downloadEunomiaData(overwrite = TRUE)
```
Comment (edward-burn) [inline tests/testthat/setup.R:6]: not sure if this would work on cran. Might be better to skip eunomia tests on cran, but run in gh actions

**Comment by edward-burn** [comment]
@catalamarti I expanded the vignette but it´s unfortunately broken the actions (as it needs the dev version of CDMConnector to be made public, but this should be imminent and when submitting PP it would work with the new CDMConnector that will be released beforehand)

File: R/addConceptIntersect.R (hunk 1 above)
Comment (catalamarti) [reply R/addConceptIntersect.R:123]: hahahha I was testing, well spotted

**Pull request merged**